profile/rwahs.xml

```diff
@@ -4219,14 +4262,43 @@
         <setting name="render">select</setting>
         <setting name="doesNotTakeLocale">1</setting>
       </settings>
+      <elements>
+        <metadataElement code="CreatorType" datatype="List" list="CreatorTypes">
```

Comment:
When I moved CreatorTypes in here one of the tests didn't pick it up, which lead me to discover that some List elements were not being checked for having a valid list.